api/db/postgres.go

```diff
@@ -132,7 +132,7 @@ func (pR *PostgresRequests) FindAllDBAnalysis(
 
 // InsertDBRepository inserts a new repository into repository table.
 func (pR *PostgresRequests) InsertDBRepository(repository types.Repository) error {
-	if (types.Repository{}) == repository {
+	if "" == repository.URL || time.Time.IsZero(repository.CreatedAt) {
```

Comment:
What do you think of using 

```go
repository.URL  == ""
```

for clean coding, stead? :smile:

Reply:
Sure looks better! Pushed changes :)